src/encoded/tests/test_types_donor.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+@pytest.fixture
+def parent(testapp, award, lab, human):
+    item = {
+        'lab': lab['@id'],
+        'award': award['@id'],
+        'organism': human['@id']
+    }
+    return testapp.post_json('/human_donor', item).json['@graph'][0]
+
+
+@pytest.fixture
+def child(testapp, award, lab, human):
```

Comment:
More descriptive name.

Reply:
I changed the name to child_human_donor.